tests/piqueserver/test_badmin.py

```diff
@@ -41,4 +47,4 @@ def returnZero():
 
         badmin.reactor.seconds = returnZero
```

Comment:
`reactor.seconds` does not get restored after this. This means other tests might break. I also suggest you use the functionality of `unittest.mock` to set return values and `patch` to return `reactor.seconds` to the original value (https://docs.python.org/3/library/unittest.mock.html)